### PR TITLE
Fixed some manuals not printing on the library computer

### PR DESCRIPTION
**Pull request by DeityLink**
:cl:
* bugfix: Fixed some manuals not printing on the library computer.

### CODE REVIEW

**Comment by DamianX** [comment]
How/why does this work? Why not just use subtypesof?

**Comment by DeityLink** [comment]
dunno about that. I'm just fixing the bug. the problem was manualcount not incrementing when iterating through a forbidden manual.

if we were to change the feature, I'd remove the check for the database connection, which isn't needed to print bloody manuals.

File: code/modules/library/computers/checkout.dm [hunk 1]
```diff
@@ -177,12 +177,13 @@
...
-			for(var/manual_type in (typesof(/obj/item/weapon/book/manual) - forbidden))
-				M = new manual_type()
-				dat += "<tr><td><A href='?src=\ref[src];manual=[manualcount]'>[M.title]</A></td></tr>"
+			for(var/manual_type in typesof(/obj/item/weapon/book/manual))
+				if (!(manual_type in forbidden))
+					M = new manual_type()
+					dat += "<tr><td><A href='?src=\ref[src];manual=[manualcount]'>[M.title]</A></td></tr>"
+					qdel(M)
+					M = null
 				manualcount++
```
Comment (gbasood) [inline code/modules/library/computers/checkout.dm:186]: The placement of this line differs now and will behave differently than before

Comment (DeityLink) [reply code/modules/library/computers/checkout.dm:186]: uh?...well yeah...that's specifically what fixes the bug.

Comment (gbasood) [reply code/modules/library/computers/checkout.dm:186]: That's... weird? I don't like this. But okay.

**Pull request merged**